charts/backend/values.yaml

```diff
@@ -45,12 +45,12 @@ commonConf:
 
 
 securityContext:
-  fsGroup: 1001
-  runAsUser: 1001
+  fsGroup: 65534
+  runAsUser: 65534
 
 image:
   repository: "europe-west4-docker.pkg.dev/stakewiselabs/private/backend"
-  tag: "v2.3.2"
+  tag: "sha-dcd7e10"
```

Comment:
Let's keep only major releases in master

Reply:
@tsudmi This is version 2.4.3b, for some reason it does not pull through it, so I use a hash